third_party/amd/backend/compiler.py

```diff
@@ -176,26 +175,6 @@ def get_arg_specialization(arg, ty, **kwargs):
             ret += "S"
         return ret
 
-    @staticmethod
-    def path_to_rocm_lld():
-        # Check env path for ld.lld
-        lld_env_path = knobs.amd.lld_path
```

Comment:
nit: this knob can now be removed

https://github.com/triton-lang/triton/blob/98d23fefa78ffe84080dbfd0b22285636570a55e/python/triton/knobs.py#L446